workspaces/orchestrator/scripts/postinstall

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+mydir=`pwd`
+cd ../../ && yarn install
```

Comment:
this shouldn't run yarn install
the developer/Ci runs yarn install
and npm should run the script after yarn install is done

Reply:
this is taken from the original line in package.json

Reply:
got it. thanks